sites/fast-creator/app/configs/fluent-ui/library.fluent-ui.design-system.mapping.ts

```diff
@@ -9,9 +9,427 @@ import {
     SwatchRGB,
 } from "@fluentui/web-components";
 
-export function setupFluentUIComponentDesignSystem(element: HTMLElement) {
+export function setupFluentUIComponentDesignSystem(
+    element: HTMLElement,
+    getAllFluentComponentTags: (containingSelector?: string) => string
+) {
     element.style.setProperty("background-color", `var(${fillColor.cssCustomProperty})`);
     element.style.setProperty("color", `var(${neutralForegroundRest.cssCustomProperty})`);
+    const fontFaceStyle = document.createElement("style");
+    fontFaceStyle.innerHTML = `
+        @font-face {
+            font-family: 'Leelawadee UI Web';
+            src: url(https://static2.sharepointonline.com/files/fabric/assets/fonts/leelawadeeui-thai/leelawadeeui-light.woff2) format('woff2'),
```

Comment:
Does this download *all* of these font-files for every visit regardless of locale?

Reply:
Probably fine for now then if that is the requirement.